tests/dhcp_server/test_dhcp_server_stress.py

```diff
@@ -0,0 +1,198 @@
+        configurated_ports.append((vlan_name, gateway, net_mask, dut_ports[index], ptf_port_indexs[index],
+                                   exp_assigned_ip_ranges[index], test_xid))
+    random_one_port = random.choice(configurated_ports)
+    dut_port, ptf_port_index, expected_assigned_ip = random_one_port[3], random_one_port[4], random_one_port[5][0]
+    logging.info("expected assigned ip is %s, dut_port is %s, ptf_port_index is %s" %
+                 (expected_assigned_ip, dut_port, ptf_port_index))
+    apply_dhcp_server_config_gcu(duthost, config_to_apply)
+
+    # Select random port with packet to be sent
+    concurrency_count = stress_config[0]
+    pkts_ports = []
+    for idx in range(concurrency_count):
+        rand_one_port = random.choice(configurated_ports)
+        rand_ptf_port_index = rand_one_port[4]
+        rand_client_mac = ptfadapter.dataplane.get_mac(0, rand_ptf_port_index).decode('utf-8')
+        pkts_ports.append(
+            (
+                create_dhcp_client_packet(
+                    src_mac=rand_client_mac,
+                    message_type=DHCP_MESSAGE_TYPE_DISCOVER_NUM,
+                    client_options=[],
+                    xid=idx
+                ),
+                rand_ptf_port_index
+            )
+        )
+
+    # Put stress on DUT
+    stress_start_time = datetime.datetime.now()
+    for idx in range(len(pkts_ports)):
+        testutils.send_packet(ptfadapter, pkts_ports[idx][1], pkts_ports[idx][0])
+
+    #  Start dhcp client at background
+    ptfhost.shell("dhclient eth%s &" % ptf_port_index)
+
+    #  Continue to put stress on DUT
+    for second in range(stress_config[1]):
+        #  Skip the first sleep to make sure the stress continues
+        #  And check result immediately after the last packet sent
+        if second != 0:
+            time.sleep(1)
+        for idx in range(len(pkts_ports)):
+            testutils.send_packet(ptfadapter, pkts_ports[idx][1], pkts_ports[idx][0])
+    stress_end_time = datetime.datetime.now()
+    stress_duration = stress_end_time - stress_start_time
+    logging.info("Elasped time during send packets is %s seconds" % stress_duration.total_seconds())
+
+    # Verify client can get IP from dhcp server within threshold
+    try:
+        def has_expected_ip_assigned(ptfhost, ptf_port_index, expected_assigned_ip):
+            ip_addr_output = ptfhost.shell("ip addr show eth%s" % ptf_port_index)['stdout']
+            logging.info("Output of ip addr show eth%s is %s" % (ptf_port_index, ip_addr_output))
+            return expected_assigned_ip in ip_addr_output
+        pytest_assert(
+            has_expected_ip_assigned(ptfhost, ptf_port_index, expected_assigned_ip),
+            'client didnt get expected IP from dhcp server within expected threshold'
+        )
+    finally:
+        ptfhost.shell("dhclient -r eth%s" % ptf_port_index, module_ignore_errors=True)
+        ptfhost.shell("killall dhclient", module_ignore_errors=True)
```

Comment:
Do we clean configuration after testing?

Reply:
Yes, we have the fixture clean_dhcp_server_config_fixture in conftest.py, this fixture will clean configuration.